.github/workflows/release-downloads.yml

```diff
@@ -72,18 +71,20 @@ jobs:
     - name: Check - non Windows
       if: runner.os != 'Windows'
       run: |
-        dafny/dafny /compileVerbose:0 /compile:0 a.dfy /proverOpt:PROVER_PATH=dafny/z3/bin/z3
+        dafny/dafny /compileVerbose:0 /compile:0 a.dfy
+        ##dafny/dafny /compileVerbose:0 /compile:0 a.dfy /proverOpt:PROVER_PATH=dafny/z3/bin/z3
```

Comment:
I'm guessing we should delete this line. Would we ever want to revert to it?
Same comment on L79.

Reply:
Correct - just testing first